pages.de/windows/clip.md

```diff
@@ -0,0 +1,20 @@
+# clip
+
+> Weiterleiten von Inhalten der Befehlsausgabe in die Windows Zwischenablage.
```

Comment:
```suggestion
> Kopieren des Inputs in die Windows-Zwischenablage.
```

Reply:
In the official documentation it is written without the dash. And it's not only the input, it can be the output too, right?

Reply:
> In the official documentation it is written without the dash.

That's ok, I didn't check. Then please remove the dash from the other example descriptions.

> And it's not only the input, it can be the output too, right?

`clip` can copy its own (I'd assume empty) output to the clipboard?

Reply:
Sorry, I think I expressed myself wrong. I meant that I would not use the word `input` in German like that. So from the side of the clipboard it is of course input, but from the command point of view it would be the output that is copied. `Inhalt` would be more neutral in my opinion.